core/src/main/java/com/microsoft/applicationinsights/internal/channel/common/TransmissionNetworkOutput.java

```diff
@@ -138,8 +139,8 @@ public boolean sendSync(Transmission transmission) {
             try {
                 // POST the transmission data to the endpoint
                 request = createTransmissionPostRequest(transmission);
-                httpClient.enhanceRequest(request);
-                response = httpClient.sendRequest(request);
+                LazyHttpClient.enhanceRequest(request);
```

Comment:
httpClient.enhanceRequest if it's not static.. i'm not sure why it got changed to be static. what is the benefit of doing this?

Reply:
same as above